Author: Nick Peelman <nick@peelman.us>
Description: Debianize nagios.cfg
--- a/sample-config/nagios.cfg.in
+++ b/sample-config/nagios.cfg.in
@@ -15,7 +15,7 @@
 # for historical purposes.  This should be the first option specified
 # in the config file!!!

-log_file=@localstatedir@/nagios.log
+log_file=/var/log/nagios/nagios.log



@@ -26,33 +26,15 @@
 # if you wish (as shown below), or keep them all in a single config file.

 # You can specify individual object config files as shown below:
-cfg_file=@sysconfdir@/objects/commands.cfg
-cfg_file=@sysconfdir@/objects/contacts.cfg
-cfg_file=@sysconfdir@/objects/timeperiods.cfg
-cfg_file=@sysconfdir@/objects/templates.cfg
-
-# Definitions for monitoring the local (Linux) host
+cfg_file=@sysconfdir@/commands.cfg
 cfg_file=@sysconfdir@/objects/localhost.cfg

-# Definitions for monitoring a Windows machine
-#cfg_file=@sysconfdir@/objects/windows.cfg
-
-# Definitions for monitoring a router/switch
-#cfg_file=@sysconfdir@/objects/switch.cfg
-
-# Definitions for monitoring a network printer
-#cfg_file=@sysconfdir@/objects/printer.cfg
-
-
 # You can also tell Nagios to process all config files (with a .cfg
 # extension) in a particular directory by using the cfg_dir
 # directive as shown below:

-#cfg_dir=@sysconfdir@/servers
-#cfg_dir=@sysconfdir@/printers
-#cfg_dir=@sysconfdir@/switches
-#cfg_dir=@sysconfdir@/routers
-
+cfg_dir=@sysconfdir@/conf.d
+cfg_dir=/usr/local/nagios/conf.d



@@ -63,7 +45,7 @@
 # directly) in order to prevent inconsistencies that can occur
 # when the config files are modified after Nagios starts.

-object_cache_file=@localstatedir@/objects.cache
+object_cache_file=/var/cache/nagios/objects.cache



@@ -102,7 +84,7 @@
 # The contents of the status file are deleted every time Nagios
 #  restarts.

-status_file=@localstatedir@/status.dat
+status_file=/var/cache/nagios/status.dat



@@ -176,7 +158,7 @@
 # is created, used, and deleted throughout the time that Nagios is
 # running.

-temp_file=@localstatedir@/nagios.tmp
+temp_file=/var/cache/nagios/nagios.tmp



@@ -242,7 +224,7 @@
 # This is the directory where archived (rotated) log files should be
 # placed (assuming you've chosen to do log rotation).

-log_archive_path=@localstatedir@/archives
+log_archive_path=@LOGDIR@/archives



@@ -1091,7 +1073,7 @@
 #	strict-iso8601	(YYYY-MM-DDTHH:MM:SS)
 #

-date_format=us
+date_format=iso8601



@@ -1176,8 +1158,8 @@
 # using the $ADMINEMAIL$ and $ADMINPAGER$ macros in your notification
 # commands.

-admin_email=@nagios_user@@localhost
-admin_pager=page@nagios_user@@localhost
+admin_email=root@localhost
+admin_pager=pageroot@localhost



@@ -1287,7 +1269,7 @@
 # DEBUG FILE
 # This option determines where Nagios should write debugging information.

-debug_file=@localstatedir@/nagios.debug
+debug_file=/var/log/nagios/debug.log
